sdks/python/hatchet_sdk/hatchet.py

```diff
@@ -491,6 +491,160 @@ def inner(
 
         return inner
 
+    @overload
+    def batch_task(
+        self,
+        *,
+        name: str | None = None,
+        description: str | None = None,
+        input_validator: None = None,
+        on_events: list[str] | None = None,
+        on_crons: list[str] | None = None,
+        version: str | None = None,
+        sticky: StickyStrategy | None = None,
+        default_priority: int = 1,
+        concurrency: ConcurrencyExpression | list[ConcurrencyExpression] | None = None,
```

Comment:
not sure if we discussed this, but how does concurrency work in this world again? like if the `max_runs` is set to a smaller number than the `batch_size` here, what happens?